view/partial/form/url.dust

```diff
@@ -0,0 +1,41 @@
+
+<form data-test="url-form" action="{action}" method="post" enctype="application/x-www-form-urlencoded">
+
+	{>"partial/alert/success" success=form.url.success/}
+	{>"partial/alert/error" errors=form.url.errors/}
+
+	<div class="field field--text">
+		<label for="url-name">
+			Name
+			<span class="field__sublabel">
+				A human-readable name for the URL, this appears across the interface
+			</span>
+		</label>
+		<input id="url-name" type="text" name="name" value="{form.url.name}" required/>
+	</div>
+
+	<div class="field field--text">
+		<label for="url-address">
+			URL to appended
```

Comment:
I think `Path` (or `pathname`) is the name for the additional part of the URL?

```
{
	"href": "https://github.com/pa11y/sidekick/pull/95/files#diff-1931cf9f658d9d73def2b171fc2ed1d6",
	"origin": "https://github.com",
	"protocol": "https:",
	"host": "github.com",
	"hostname": "github.com",
	"pathname": "/pa11y/sidekick/pull/95/files",
	"search": "",
	"hash": "#diff-1931cf9f658d9d73def2b171fc2ed1d6"
}
```

Reply:
Good point - I was using the "URL" nomenclature as this is what each of the paths are being referred to, but I guess baseUrl + path = URL.   Happy to rename!